### PR TITLE
Agnostic documentation editing tools - Closes #117

### LINKED ISSUE #117: Make documentation tools agnostic
On many places in our documentation we referred to edit a file using `vim`. Although its pretty direct way to tell someone, but some people do follow the documentation strictly as holy guideline. They may try to use `vim` but never knew how to exit from it. :) 

We should prefer to use platform and user preferences agnostic statements. Just stating that *Edit a particular file* is enough to make user understand the step, leaving the flexibility of the choice to user, that which tool he prefer to use. 

e.g. When we are saying:

```
For Docker distributions of Lisk Core, do the following to add the environment variable:

 cd lisk_repo
 vim docker/docker-compose.override.yml

Add your license key to `docker-compose.override.yml` like so:

```

We should prefer to say:

```
For Docker distributions of Lisk Core, we  need to add the environment variable to the file `lisk_repo/docker-compose.override.yml`. Afterwards it would look like: 

```

https://github.com/LiskHQ/lisk-docs/pull/112/files?utf8=✓&diff=unified#r234539032

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #117

### CODE REVIEW

File: setup/binary/binary.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Next, go to the logrotate config directory and create a new logrotate file for L
 
 ```bash
 cd /etc/logrotate.d
-vim lisk
+touch lisk
 ```
```
Comment (lsilvs) [inline setup/binary/binary.md:136]: I would change the sentence above and remove these commands.

Something like:
```
Next, create a new file called `lisk` in the logrotate directory `/etc/logrotate.d` with the following content:
```

File: setup/source/source.md [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
   2. [Create a new user](#create-a-new-user)
   3. [Install Tool chain components](#tool-chain-components)
-  4. [Git Installation](#git)
-  5. [Node.js Installation](#node-js)
-      1. [Recommended: Install NVM ](#5a-recommended-install-a-version-manager-such-as-nvm-httpsgithubcomcreationixnvm)
-      2. [Recommended: Install PM2 ](#5b-recommended-pm2-httpsgithubcomunitechpm2)
-  6. [Postgres Installation](#postgresql-version-10)
-  7. [Redis Installation](#installing-redis)
+  4. [Git](#git)
+  5. [Node.js](#node-js)
```
Comment (lsilvs) [inline setup/source/source.md]: This anchor is not working

File: setup/source/source.md [hunk 3]
```diff
@@ -330,7 +336,7 @@ Next, go to the logrotate config directory and create a new logrotate file for L
 
 ```bash
 cd /etc/logrotate.d
-vim lisk
+touch lisk
 ```
```
Comment (lsilvs) [inline setup/source/source.md:340]: Same here.

File: user-guide/administration/source/admin-source.md [hunk 4]
```diff
@@ -12,59 +23,58 @@ This section details how to manage a Source installation of Lisk Core. Source in
...
+Restart Lisk Core.
+```bash
 pm2 restart lisk
 ```
 
-Delete Source Installation with PM2 (useful if removing Lisk Core or rebuilding the source installation entirely)
-
-```shell
+### Delete
+Will remove Lisk Core process from pm2 list.
```
Comment (lsilvs) [inline user-guide/administration/source/admin-source.md]:
```suggestion
Remove Lisk Core process from pm2 list.
```

File: user-guide/monitoring/monitoring.md [hunk 5]
```diff
@@ -42,14 +42,13 @@ export NEW_RELIC_LICENSE_KEY=XXXXXXXXX
 
 ##### Docker
 
-For Docker distributions of Lisk Core, do the following to add the environment variable:
+For Docker distributions of Lisk Core navigate into the `docker` folder inside your Lisk Core installation:
 
 ```bash
-cd lisk_repo
-vim docker/docker-compose.override.yml
+cd lisk_repo/docker # navigate into docker directory
```
Comment (lsilvs) [inline user-guide/monitoring/monitoring.md:48]: This is unnecessary imo

File: user-guide/monitoring/monitoring.md [hunk 6]
```diff
@@ -91,12 +85,7 @@ If you use "vim", press `i` to get into the insert mode.
 license_key: 'XXXXXXXXX',
 ```
 
-After adding the license key, save changes and quit the editor:
-
-```bash
-<esc> # press esc to quit insert mode of vim
-:wq # save changes and quit vim afterwards
-```
+After adding the license key, save your changes and exit the file.
```
Comment (lsilvs) [inline user-guide/monitoring/monitoring.md]:
```suggestion
After adding the license key, save your changes and reload your node.
```

**Review by lsilvs — CHANGES_REQUESTED** [review]

File: setup/source/source.md (hunk 2 above)
Comment (nazarhussain) [inline setup/source/source.md]:
```suggestion
  5. [Node.js](#nodejs)
```

File: setup/binary/binary.md (hunk 1 above)
Comment (nazarhussain) [reply setup/binary/binary.md:136]: Although @lsilvs suggestion makes it completely agnostic, but I believe having such simple commands as `cd` and `touch` is not bad. These are kind of operating system commands, and every user must be aware of these. Also that's the easiest way to communicate such tasks to user. 

May be we add text description as well but keep the script there.

File: user-guide/administration/source/admin-source.md [hunk 7]
```diff
@@ -12,59 +23,58 @@ This section details how to manage a Source installation of Lisk Core. Source in
...
-pm2 start --name lisk app.js
-```
-
-After the process is started, its runtime status and log location can be retrieved by issuing the following command:
-
-```shell
+### Status
+Check the status of the Lisk Core Node.
+```bash
 pm2 show lisk
```
Comment (nazarhussain) [inline user-guide/administration/source/admin-source.md]:
```suggestion
pm2 status lisk
```

**Review by nazarhussain — CHANGES_REQUESTED** [review]

**Review by lsilvs — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**